pysam/libcbcf.pyx

```diff
@@ -1184,6 +1185,15 @@ cdef inline bcf_sync_end(VariantRecord record):
         bcf_info_set_value(record, b'END', record.ptr.pos + record.ptr.rlen)
 
 
+cdef inline bint has_symbolic_allele(VariantRecord record):
```

Comment:
change return type to be `int` and return the index of the first symbolic allele or zero if none exist.  This preserves the boolean nature of the function (0==no symbolic alleles, !=0 is the index if the first) and provides a more informative return type.